test/Macros/macro_default_argument.swift

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: swift_swift_parser
+
+// RUN: %empty-directory(%t)
+// RUN: %host-build-swift -swift-version 5 -emit-library -o %t/%target-library-name(MacroDefinition) -module-name=MacroDefinition %S/Inputs/syntax_macro_definitions.swift -g -no-toolchain-stdlib-rpath
+
+// RUN: not %target-build-swift -typecheck -swift-version 5 -load-plugin-library %t/%target-library-name(MacroDefinition) %s 2>&1 | %FileCheck %s
+
+@freestanding(expression)
+macro MagicLine() -> Int = #externalMacro(module: "MacroDefinition", type: "MagicLineMacro")
+
+struct LineContainer {
+    let line: Int
+}
+
+func partOfDefaultArgumentOkay(container: LineContainer = .init(line: #MagicLine)) {
+    print(container.line)
+}
+
+func parenthesizedExpansionAtDeclOkay(line: Int = (#MagicLine)) {
+    print(line)
+}
+
+func builtInOkay(line: Int = #line) {
+    print(line)
+}
+
+// CHECK: non-built-in macro cannot be used as the default argument
+func asDefaultArgument(line: Int = #MagicLine) {
```

Comment:
Can you use `-verify` on the command line so this `CHECK:` and `FileCheck` usage can be replaced with:
```
// expected-error{{non-built-in macro cannot be used as the default argument}}
```

Reply:
Thanks for the suggestion! Updated the test file to use `-verify`